pkg/resources/common.go

```diff
@@ -101,3 +104,45 @@ func ImportName[T sdk.AccountObjectIdentifier | sdk.DatabaseObjectIdentifier | s
 
 	return []*schema.ResourceData{d}, nil
 }
+
+func CommonImportWrapper(resourceName resources.ResourceName, importImplementation schema.StateContextFunc) schema.StateContextFunc {
```

Comment:
I think I would leave them here, it makes sense to group them with things that are already present here. I was looking for a better place like helpers for resources, but we have this for now. Maybe with common delete function etc. it could be moved to a resource_helpers.go (or something similar) file.